docs/content/4.api/1.components/7.prose.md

```diff
@@ -70,6 +70,10 @@ Component properties will be:
 
 Check out the [highlight options](/api/configuration#highlight) for more about the syntax highlighting.
 
+::alert{type="warning"}
+If you want to use `]` in the filename, you need to escape it with 2 backslashes: `\\]`. This is necessary since JS will automatically escape the backslash in a string so `\]` will be resolved as `]` breaking our regex.
```

Comment:
Is the rendered Markdown for `\]` and `\\] ` properly rendered, so we see the backslash(es)?
I think in code blocks we don't have to escape, but I'm not sure...